Library/Formula/llvm.rb

```diff
@@ -198,12 +196,6 @@ def install
 
     args << "-DLLVM_ENABLE_RTTI=On" if build.with? "rtti"
 
-    if build.with? "assertions"
-      args << "-DLLVM_ENABLE_ASSERTIONS=On"
-    else
-      args << "-DCMAKE_CXX_FLAGS_RELEASE='-DNDEBUG'"
```

Comment:
I think this argument was added for a reason for the “without assertions” case. Are you sure it is not needed in release mode (because at some point it was) and can be safely removed?